components/brave_ads/core/internal/database/database_manager.cc

```diff
@@ -19,6 +21,16 @@
 
 namespace brave_ads {
 
+namespace {
+
+void DoDumpWithoutCrashing(const int from_version, const int to_version) {
```

Comment:
nit: could we please raise an issue to remove this at a later date and add a `// TODO` comment with the issue number etc.